src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+  
+    <form action="#" method="post">
+      <div class="input-field" data-qa="big">
+        <span class="input-img-big"></span>
+        <label for="bigForm"></label>
+        <input 
+          class="input-form bigForm" 
```

Comment:
you don't need both small form and big form, let input-form be our default form (big) and then you will modify some styles for smallForm

```suggestion
          class="input-form" 
```

Reply:
done